main.tf

```diff
@@ -8,6 +8,7 @@ moved {
 }
 
 resource "tls_private_key" "ssh" {
+  count = var.admin_username == null ? 0 : 1
```

Comment:
We have a format issue here, the space is no aligned, would you please run `terraform format` on this file? Thanks!

Reply:
Done